tnt/src/main/java/org/dcsa/conformance/standards/tnt/party/SortCriteria.java

```diff
@@ -0,0 +1,18 @@
+package org.dcsa.conformance.standards.tnt.party;
+
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+public class SortCriteria {
+  private final String field;
+  private final List<AttributeMapping> attributeMappings;
+  private final SortDirection direction;
```

Comment:
I think you can use the Java Record keyword here as well. You don't need the constructor then. And records are automatically immutable. See e.g. https://www.baeldung.com/java-record-keyword

Reply:
done